rst/topics/failover.rst

```diff
@@ -274,7 +274,19 @@ In Lua API:
 
 You could also make node unelectable in WebUI:
```

Comment:
I also suggest the following change:

> You can restrict node to be elected in ``stateful`` failover mode by GraphQL
> or Lua API. "Unelectable" node can't become a leader in replicaset.

You can restrict the election of a particular node in the ``stateful``failover mode by GraphQL or Lua API. An "unelectable" node can't become a leader in a replicaset.